pkgs/development/haskell-modules/hoogle.nix

```diff
@@ -63,10 +63,11 @@ buildPackages.stdenv.mkDerivation {
   passAsFile = ["buildCommand"];
 
   buildCommand = ''
-    ${lib.optionalString (packages != [] -> docPackages == [])
+    ${let packages' = lib.filter (p: p != null) packages;
```

Comment:
Could you add a comment here (maybe linking to the issue in question) that explains why we are explicitly filtering out `null`?

Reply:
done!